.github/workflows/i18n-clean-up.yml

```diff
@@ -0,0 +1,18 @@
+name: Clean up after Crowdin download
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  delete-l10n-branch-after-merge:
```

Comment:
We use i18n instead of l10n - please update all mentions of l10n in this PR to i18n 😌